scripts/st2bootstrap-el7.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+set -eux
```

Comment:
We should definitely fix this.

This was one of the more annoying parts of existing bootstrap script - even if some step or command failed we still tried to continue which caused a lot of confusion and unnecessary support burden.

We should fail loud and early.